### PR TITLE
Avoid accessing canvas object if not necessary since some users may p…

### PR DESCRIPTION
**Pull request by juj**
…ass null on context creation when Module['preinitializedWebGLContext'] is used. #13926

### CODE REVIEW

**Review by MarkCallow — APPROVED** [review]

**Review by MarkCallow — APPROVED** [review]
Changes look good to me. I don't have write access so my review is not enough to unblock the merge. Somebody with write permissions please do it.

File: src/library_webgl.js
Comment (kripken) [inline src/library_webgl.js:633]: I'm missing something here. This moves code that uses `canvas`. But that variable is not change in any of the code we skipped over. How is the change noticeable?

Comment (MarkCallow) [reply src/library_webgl.js:633]: I can't figure out how to make a link to the relevant lines but look at lines (post change) 606 to 618 in `library_webgl.js`. Line 610 has `if (Module['preinitializedWebGLContext']) {` and line 617 just before the moved code has the matching `} else {`. So the code is no longer executed if there a pre-initialized context is being used.

Comment (kripken) [reply src/library_webgl.js:633]: Thanks, I see, so it's in the `else`. The lack of indentation confused me. (But it's correct to not be indented, given that the if is ifdefed.)

**Review by kripken — APPROVED** [review]

**Pull request merged**
